deploy/aws-account-shredder-template.yaml

```diff
@@ -15,6 +15,44 @@ parameters:
     value : "1"
 
 objects:
+  - apiVersion: v1
+    kind: ServiceAccount
+    metadata:
+      name: aws-account-shredder
+      namespace: aws-account-shredder
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: Role
+    metadata:
+      namespace: aws-account-shredder
+      name: aws-account-shredder
+    rules:
+      - apiGroups:
+          - route.openshift.io
+        resources:
+          - routes
+        verbs:
+          - '*'
+      - apiGroups:
+          - ""
+        resources:
+          - services
+          - secrets
+        verbs:
+          - '*'
+  - apiVersion: rbac.authorization.k8s.io/v1
+    kind: RoleBinding
+    metadata:
+      name: aws-account-shredder
+      namespace: aws-account-shredder
+    subjects:
+      - kind: ServiceAccount
+        name: aws-account-shredder
+        namespace: aws-account-shredder
+    roleRef:
+      kind: Role
+      name: aws-account-shredder
+      apiGroup: rbac.authorization.k8s.io
+      namespace: aws-account-shredder
```

Comment:
This could be removed as it is in the same namespace

Reply:
removed hard-coded namespace , but keeping this variable ( passing by parameter ) . once the shredder gets deployed i will go ahead and remove it.